README.md

```diff
@@ -54,39 +54,44 @@ sudo yum install openssl libcurl libxml2 pinentry xclip openssl-devel libxml2-de
 
 
 #### Debian/Ubuntu
+
 * Install the needed build dependencies, and then follow instructions in
   the 'Building' section.
 
-* For Debian:
+* For Debian (stable/oldstable) and Ubuntu < 18.04
 
 ```
-sudo apt install --no-install-recommends \
+apt-get --no-install-recommends -yqq install \
+  bash-completion \
   build-essential \
   cmake \
   libcurl3  \
-  libcurl4-openssl-dev  \
-  libssl-dev  \
-  libxml2  \
+  libcurl3-openssl-dev  \
+  libssl1.0 \
+  libssl1.0-dev \
+  libxml2 \
   libxml2-dev  \
-  openssl  \
-  pinentry-curses \
-  pkg-config  \
-  xclip
+  pkg-config \
+  ca-certificates \
+  xclip \
```

Comment:
Trailing slash is not needed in the last line.

Reply:
Fixed.